flw/src/main/java/org/motechproject/nms/flw/web/FrontLineWorkerImportController.java

```diff
@@ -69,11 +75,14 @@ public void importFrontLineWorkers(@RequestParam MultipartFile csvFile) {
         try {
             try (InputStream in = csvFile.getInputStream()) {
                 frontLineWorkerImportService.importData(new InputStreamReader(in));
+                csvAuditService.auditSuccess(csvFile.getName(), "/flw/import");
```

Comment:
Should this call move into the logError method?  I may have added some catch statements with my most recent PR so moving this into there would insure that the audit log is still written (since I assume I'm calling logError)